Makefile

```diff
@@ -371,3 +371,19 @@ update-watcher-csv:
 	if [ "$(has_webhooks)" != "null" ]; then \
 	    oc patch -n $(OPERATOR_NAMESPACE) $(csv) --type json -p='[{"op": "remove", "path": "/spec/webhookdefinitions"}]'; \
 	fi
+
+CATALOG_IMAGE ?= quay.io/openstack-k8s-operators/watcher-operator-index:latest
+
+.PHONY: watcher
+watcher: export CATALOG_IMG=${CATALOG_IMAGE}
+watcher: ## Deploying watcher operator via olm
+	bash ci/olm.sh
+	oc apply -f ci/olm.yaml
+	$(eval csvname=$(shell oc get csv -n openstack-operators -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}' | grep -E "^watcher-operator\.v"))
+	timeout 30 bash -c 'until [ "$(shell oc get -n openstack-operators csv/$(csvname) -o jsonpath='{.status.phase}')" == "Succeeded" ]; do sleep 5; done'
```

Comment:
This is failing in my test:

`$ make watcher
error: arguments in resource/name form must have a single resource and name
bash ci/olm.sh
oc apply -f ci/olm.yaml
namespace/openstack-operators created
catalogsource.operators.coreos.com/watcher-operator-index created
operatorgroup.operators.coreos.com/openstack created
subscription.operators.coreos.com/watcher-operator created
timeout 30 bash -c 'until [ "" == "Succeeded" ]; do sleep 5; done'
make: *** [Makefile:383: watcher] Error 124
`

I'd say it's not related to timeout, i tested with a much longer one and get the same issue.

Reply:
Thank you @amoralej for testing it! I realized I also hit the same issue in multiple runs. I have fixed it now.

Reply:
@cescgina Thank you for the review. Can you try now?

Reply:
@raukadah thanks, the last version works perfectly!